src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import { Post } from '../../types/Post';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {(user)
+            ? <UserInfo user={user} />
+            : 'Oops... Unknown user'}
```

Comment:
))) cool)) but seems if no user we don't have to show anything

Reply:
I agree with you)) I think, every post will have its user, and there is no sense in condition. But in my function getUser return value is `User | null`, cause method FIND can return undefined... And I don't know how could I cheat it))